source/scale/scale-to-200000-users.rst

```diff
@@ -0,0 +1,64 @@
+Scale Mattermost up to 200000 users
+====================================
+
+.. include:: ../_static/badges/ent-selfhosted.rst
+  :start-after: :nosearch:
+
+This page describes the Mattermost reference architecture designed for the load of up to 200000 concurrent users. Unsure which reference architecture to use? See the :doc:`scaling for enterprise </scale/scaling-for-enterprise>` documentation for details.
+
+- **High Availability**: Required
+- **Database Configuration**: writer, multiple readers
+
+.. note::
+  - Usage of CPU, RAM, and storage space can vary significantly based on user behavior. These hardware recommendations are based on traditional deployments and may grow or shrink depending on how active your users are.
+  - From Mattermost v10.4, Mattermost Enterprise customers can configure `Redis <https://redis.io/>`_ (Remote Dictionary Server) as an alternative cache backend. Using Redis can help ensure that Mattermost remains performant and efficient, even under heavy usage. See the :ref:`Redis cache backend <configure/environment-configuration-settings:redis cache backend>` configuration settings documentation for details.
```

Comment:
Do we need the Remote Dictionary Server parenthesis?
```suggestion
  - From Mattermost v10.4, Mattermost Enterprise customers can configure `Redis <https://redis.io/>`_ as an alternative cache backend. Using Redis can help ensure that Mattermost remains performant and efficient, even under heavy usage. See the :ref:`Redis cache backend <configure/environment-configuration-settings:redis cache backend>` configuration settings documentation for details.
```

Reply:
I don't know, this was already there. Will defer to Carrie.